cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -1014,7 +1018,12 @@ impl Inst {
                 let rd = format_reg(rd.to_reg(), allocs);
                 format!("{src1} + {src2} => {rd};")
             }
-
+            Inst::Ineg { rd, rs1 } => {
+                let rd = format_reg(rd.to_reg(), allocs);
+                let rs = format_reg(*rs1, allocs);
+                // FIXME: should this use a SUB?
+                format!("0 - {rs} => {rd}")
```

Comment:
Maybe better to directly write that it is ``Ineg``? At least something like ``0 - {rs} => {rd}   # Ineg generated``

Reply:
FWIW I think it would be great if vcode looked reasonably close to the real code, instead of being some debug-like representation of the internal data structures. At the very least it seems like that’s the case for other architectures currently.